src/main/java/calculator/InputValidator.java

```diff
@@ -0,0 +1,17 @@
+package calculator;
+
+public class InputValidator {
+
+    public void validateInput(String input) {
+        if (!input.matches("\\d+")) {
+            throw new IllegalArgumentException("잘못된 값 입력됨. 다시 실행하시오.");
+        }
+    }
+
+    public int positiveNumber(Integer input) {
+        if (input <= 0) {
+            throw new IllegalArgumentException("양수 외에 불가. 다시 실행하시오.");
+        }
+        return input;
```

Comment:
"" 안의 문구 같은 경우 constant 클래스나 Enum 클래스로 따로 관리하시는게 좋을 것 같습니다.
constant 클래스 예시
```suggestion
        public class Constants {
    public static final String POSITIVE_NUMBER_ONLY = "양수 외에 불가. 다시 실행하시오.";
    public static final String INVALID_INPUT = "잘못된 값 입력됨. 다시 실행하시오.";
}
```

enum 클래스 예
```suggestion
        public enum ErrorMessage {
    POSITIVE_NUMBER_ONLY("양수 외에 불가. 다시 실행하시오."),
    INVALID_INPUT("잘못된 값 입력됨. 다시 실행하시오.");

    private final String message;

    ErrorMessage(String message) {
        this.message = message;
    }

    public String getMessage() {
        return message;
    }
}
```
이렇게 관리하시면 추후 변경사항이 생겨서 문구를 변경해야 할 때 단순히 constant 클래스나 enum 클래스로 가서 문구만 변경하면 위 문구를 사용하는 여러 곳에서 바꿀 필요가 없어지므로 유지보수 측면에서 이점이 있습니다.